src/chain_service/faucet.ts

```diff
@@ -0,0 +1,236 @@
+import { ApiPromise, WsProvider, Keyring } from "@polkadot/api";
+import { checkAddress, cryptoWaitReady } from "@polkadot/util-crypto";
+import BN from "bn.js";
+import { ApiManager, API } from "./api.js";
+import { Config } from "../config.js";
+import {
+  AddressError,
+  DispatchError,
+  NoFundsError,
+  RateError,
+} from "./faucetErrors.js";
+import { SlackNotifier } from "../slack_service/slack.js";
+
+const customTypes = {
```

Comment:
This doesn't seem to be used.

Reply:
Not really, no. There were a legacy code from pendulum-faucet, I assumed it could be important. But I just checked and it is only important if we needed to override the default types. 
Since we don't even use those return types, I agree is better to remove them.